hudi-client/hudi-java-client/src/main/java/org/apache/hudi/client/common/HoodieJavaEngineContext.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.client.common;
+
+import org.apache.hadoop.conf.Configuration;
```

Comment:
nit: import needs to be reordered

Reply:
I am not sure about it, when I run "mvn checkstyle:checkstyle", there is no error message. Is there any way to find out where the import order is wrong.

Reply:
I couldn't find the import ordering from docs anymore, probably being rephrased or somehow. The checkstyle and spotless are not mature enough at this point https://issues.apache.org/jira/browse/HUDI-1237, so I personally reformating the code myself by looking at other codes. Apologize for the confusion.